sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastSuite.scala

```diff
@@ -971,11 +971,7 @@ abstract class AnsiCastSuiteBase extends CastSuiteBase {
         checkCastWithParseError("20150318")
         checkCastWithParseError("2015-031-8")
         checkCastWithParseError("2015-03-18T12:03:17-0:70")
-
-        val input = "abdef"
-        checkExceptionInExpression[DateTimeException](
-          cast(input, TimestampType, Option(zid.getId)),
-          s"Cannot cast $input to TimestampType.")
+        checkCastWithParseError("abdef")
```

Comment:
It doesn't need to make a followup only to fix some nits. We can just do it when we happen to touch the codes around here.

Reply:
I am going to approve this PR for this time since it's already open but let's avoid next time.

Reply:
OK. I got it, will try my best to avoid such issue in future.